Pages/fashionCategoryPage.js

```diff
@@ -0,0 +1,31 @@
+class FashionCategoryPage{
```

Comment:
You should have parent "Category" page.
And then if needed, add child pages for different categories.

Reply:
Instead of Category page, I added Page page, which containts methods that certain Category pages (for example FashionCattegoryPage) inherits.

Reply:
Page page object should contain getters and method which are shared between all page objects (something which is always present and usable in every page-object in appliation, general methods, example openLink ..)

Category page object should contain something which is shared between all categories (for example you have methods "clickOnItem" and "waitForProductItem" in fashion category page-object, but those method could be used also for every other category ? That is why it should be inside Category page object class).
And then if there is something specific only for fashion category, you can write that part inside fashion category page object.

Reply:
I added Category class, which extends Page class, and FashionCattegory class extends Category class

Reply:
Why do you have the same methods in different page objects, for example
<img width="601" alt="Screenshot 2020-11-19 at 10 24 10" src="https://user-images.githubusercontent.com/50828644/99647008-7e066d00-2a51-11eb-926e-87b7b5ccc0be.png">
in HomePage and Page ?
If methods are doing the same job, there should be only one implementation.
This repeats in code.

Reply:
I removed methods that consists of only calling super.method()

Reply:
Please remove those getters as well.

Reply:
I did my best to remove getters which were duplicates in page objects